Runtime/Scripts/BehaviorTree/Decorators.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using RegressionGames.RGBotLocalRuntime;
+
+namespace RegressionGames.BehaviorTree
+{
+    /// <summary>
+    /// A node that always returns <see cref="NodeStatus.Success"/> when executed.
```

Comment:
The decorators should clarify that they alter/override their child's status

Reply:
Ah, I see now that the `AlwaysFail` and `AlwaysSucceed` nodes should still accept one child. I'll make that change and update the docs. Currently they cannot accept any child nodes.